src/store/mutations.js

```diff
@@ -0,0 +1,16 @@
+import _ from 'lodash'
+
+export default {
+  UPDATE_LIST (state, payload) {
```

Comment:
should this be separated out into separate add/remove mutations?

Reply:
I was thinking this was more DRY in the template and as a mutation. But after more thought, I don't think it's as explicit and it would be helpful if you were debugging to have two mutations.

Reply:
See 05d4625